server/index.js

```diff
@@ -45,6 +45,14 @@ app.get('/test', (req, res) => {
   res.render('widget-test');
 });
 
+app.get('/kb-test', (req, res) => {
```

Comment:
We don't test widgets on it's own server. We need to test it on `erxes-target-app`.

Reply:
how do i setup that erxes-target-app?